cirq-core/cirq/ops/fsim_gate.py

```diff
@@ -347,6 +347,35 @@ def from_fsim_rz(
         chi = (b0 - b1 - a0 + a1) / 2.0
         return PhasedFSimGate(theta, zeta, chi, gamma, phi)
 
+    @staticmethod
+    def from_matrix(u: np.ndarray) -> Optional['PhasedFSimGate']:
```

Comment:
Please add docstring, it is a new public function.

Reply:
done